modules/pensions/app/models/pensions/saved_claim.rb

```diff
@@ -85,6 +87,38 @@ def send_confirmation_email
           'confirmation_number' => guid
         }
       )
+
+      insert_notification
+    end
+
+    ##
+    # insert notifcation after VANotify email send
+    #
+    # @see ClaimVANotification
+    #
+    def insert_notification
+      ClaimVANotification.create!(
+        form_type: form_id,
+        saved_claim_id: id,
+        email_sent: true,
+        email_template_id: Settings.vanotify.services.va_gov.template_id.form527ez_confirmation_email,
```

Comment:
should probably make this an arg to the function, so can be used for additional templates